launching-software/infrastructure.md

```diff
@@ -119,7 +117,7 @@ except Identity and Access Management (IAM).
   - [COE](https://docs.google.com/forms/d/e/1FAIpQLSdn8Pjof5xWZSkopkEU6pumQlXGYfU4fMN9efmoDzffm2MW4A/viewform)
 - [Sign in to your sandbox account](https://tts-sandbox.signin.aws.amazon.com/console)
 
-#### Rules
+**Rules**
```

Comment:
I don't think we want to close this issue while leaving the headings un-navigable. Maybe we can sync up so I can better understand the issue?